src/_pytest/_code/code.py

```diff
@@ -391,17 +391,70 @@ def recursionindex(self):
 )
 
 
+@attr.s(repr=False)
 class ExceptionInfo(object):
     """ wraps sys.exc_info() objects and offers
         help for navigating the traceback.
     """
 
-    _striptext = ""
     _assert_start_repr = (
         "AssertionError(u'assert " if _PY2 else "AssertionError('assert "
     )
 
-    def __init__(self, tup=None, exprinfo=None):
```

Comment:
Is `exprinfo` and `_striptext` the same thing? I ask because we are changing the signature of the init method.

Reply:
the signature change is valid due to the class being a internal